wp-cldr.php

```diff
@@ -10,6 +10,138 @@
  * Domain Path: /languages/
  * License:     GPLv2 or later
  * License URI: https://www.gnu.org/licenses/gpl-2.0.html
+ *
+ * @package    wp-cldr
  */
 
 require_once plugin_dir_path( __FILE__ ) . 'class.wp-cldr.php';
+require_once( ABSPATH . 'wp-admin/includes/translation-install.php' );
+
+add_action( 'admin_menu', 'wp_cldr_menu' );
+
+/**
+ * Gets the wp-admin menu info.
+ */
+function wp_cldr_menu() {
+	add_options_page(
+		'WP CLDR Options',
+		'WP CLDR',
+		'manage_options',
+		'wp-cldr',
+		'wp_cldr_settings'
+	);
+}
+
+/**
+ * Gets the settings and examples page for the plugin.
+ */
+function wp_cldr_settings() {
+
+	if ( isset( $_GET['locale'] ) && 1 < strlen( sanitize_text_field( wp_unslash( $_GET['locale'] ) ) ) ) {
+		$locale = sanitize_text_field( wp_unslash( $_GET['locale'] ) );
+	} else {
+		$locale = get_locale();
+	}
+	if ( isset( $_GET['country'] ) && 2 === strlen( sanitize_text_field( wp_unslash( $_GET['country'] ) ) ) ) {
+		$country = sanitize_text_field( wp_unslash( $_GET['country'] ) );
+	} else {
+		$country = 'US';
+	}
+	$cldr = new WP_CLDR( $locale );
+	$locales = $cldr->languages_by_locale();
+	$languages = get_available_languages();
+	$translations = wp_get_available_translations();
+	$territories = $cldr->territories_by_locale();
+	?>
+
+	<div class="wrap">
+	<h1><?php esc_html_e( 'WP CLDR settings and info' ); ?></h1>
+	<?php _e( 'This plugin provides WordPress developers with easy access to localized country/region names, language names, currency names/symbols/usage, and other localization info from the <a href="http://cldr.unicode.org/">Unicode Common Locale Data Repository (CLDR)</a>. Here are some examples:', 'wp-cldr' ) . '<br>'; ?>
```

Comment:
* Output should be escaped using `esc_html_e`
* In general, HTML should not be present in strings to translate. It's better to split it into multiple calls to the translation function.